conans/test/functional/scm/scm_test.py

```diff
@@ -52,7 +60,7 @@ def _commit_contents(self):
         self.client.runner("git add .", cwd=self.client.current_folder)
         self.client.runner('git commit -m  "commiting"', cwd=self.client.current_folder)
 
-    def test_scm_other_type_ignored(self):
+    def test_scm_other_type_ignored(self):  # FIXME: is this test ok?
```

Comment:
I really wanted an opinion about this test. Having `scm = ["Other stuff"]` should be ok? Why not raise? And what should be the output after the SCM replacement? cc/ @danimtb @memsharded

Reply:
Ok, I've modified the test to check that the value is actually preserved.